rocky/katalogus/views/mixins.py

```diff
@@ -103,7 +103,16 @@ def run_boefje(self, katalogus_boefje: KATalogusBoefje, ooi: Optional[OOI]) -> N
         )
 
         item = QueuePrioritizedItem(id=boefje_task.id, priority=1, data=boefje_task)
-        client.push_task(f"boefje-{self.organization.code}", item)
+        try:
+            client.push_task(f"boefje-{self.organization.code}", item)
+        except Exception as e:
+            messages.add_message(
+                self.request,
+                messages.ERROR,
+                e,
```

Comment:
This doesn't seem right. Have you tested this?

Reply:
Yes the message returned by the exception is displayed as a message, 2nd screenshot